ppr-ui/src/resources/registrationTypes.ts

```diff
@@ -315,6 +315,44 @@ export const MhrRegistrationType: RegistrationTypeIF = {
   text:
     `${UIRegistrationTypes.MANUFACTURED_HOME_REGISTRATION} (${APIRegistrationTypes.MANUFACTURED_HOME_REGISTRATION})`
 }
+export const MHRegistrationTypes: Array<MhRegistrationTypeIF> = [
+  {
+    class: 'registration-list-header',
+    disabled: true,
+    divider: false,
+    group: 3,
+    registrationTypeUI: null,
+    registrationTypeAPI: null,
+    text: 'Standard Registrations'
+  },
+  {
+    class: 'registration-list-item',
+    disabled: false,
+    divider: false,
+    group: 3,
+    registrationTypeUI: APIMhrDescriptionTypes.REGISTER_NEW_UNIT,
+    registrationTypeAPI: APIMhrDescriptionTypes.REGISTER_NEW_UNIT,
+    text: `${APIMhrDescriptionTypes.REGISTER_NEW_UNIT} (${APIMhrDescriptionTypes.REGISTER_NEW_UNIT})`
+  },
+  {
+    class: 'registration-list-item',
+    disabled: false,
+    divider: false,
+    group: 3,
+    registrationTypeUI: APIMhrDescriptionTypes.SALE_OR_GIFT,
+    registrationTypeAPI: APIMhrDescriptionTypes.SALE_OR_GIFT,
+    text: `${APIMhrDescriptionTypes.SALE_OR_GIFT} (${APIMhrDescriptionTypes.SALE_OR_GIFT})`
+  },
+  {
+    class: 'registration-list-item',
+    disabled: false,
+    divider: false,
+    group: 3,
+    registrationTypeUI: APIMhrDescriptionTypes.CONVERTED,
+    registrationTypeAPI: APIMhrDescriptionTypes.CONVERTED,
+    text: `${APIMhrDescriptionTypes.CONVERTED} (${APIMhrDescriptionTypes.CONVERTED})`
```

Comment:
As mentioned in the root of the pr, this is probably where we could map the display value from `** Converted **` etc

Reply:
** Converted  ** does work.    Was previously taken care of in the types..

Reply:
@ChasBoucher 
For sure it works, i'm speaking to the display value of it. See the screenshot above. 
Design might not want `** CONVERTED **` displayed in the dropdown. 
In the table we mapped to simply `Converted`, might have to do the same here.

Reply:
![image](https://user-images.githubusercontent.com/53542131/210428203-37d32f7a-1f21-4f0f-800b-ac821f9ca345.png)

Reply:
I see...

Reply:
I made the change(add UIMhrDesctiptiontypes).
Found issue with SALE OR GIFT and fixed.